demos/CMakeLists.txt

```diff
@@ -2,14 +2,38 @@
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_EXTENSIONS off)
 
-# Enable at least some optimization in all builds. The Ret2Spec demo, in
-# particular, will segfault if built without optimizations.
-# TODO(https://git.io/JecmX): Fix the crash in Ret2Spec
-add_compile_options(-O2)
-
-# Disable run-time code checking on MSVC. It's enabled by default in CMake's
-# debug build settings, but it's incompatible with optimizations.
-string(REPLACE "/RTC1" "" CMAKE_CXX_FLAGS_DEBUG ${CMAKE_CXX_FLAGS_DEBUG})
+# Enable (or at least don't disable) some optimizations in all builds.
+# Without optimizations, it can be difficult to fit enough useful code into the
+# speculative execution window.
+if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
```

Comment:
These changes were necessary in a previous incarnation to get things working in MSVC. They might not be strictly necessary anymore, but I'd like to add them anyway since it removes behavior that surprised me -- `__forceinline` wasn't actually working in MSVC!

Reply:
Thanks for discovering that.